state.go

```diff
@@ -13,4 +13,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Annotations holds runtime-specified information that is not structured by this specification.
```

Comment:
I can't find out other users of annotations right now, can you show me the use case? absolutely you can read it, but right now only runtime stores information in it.